src/components/Insight/InsightCardSmall.module.scss

```diff
@@ -3,10 +3,9 @@
   font-size: 16px;
   line-height: 24px;
   color: var(--mirage);
-  max-width: 100%;
-  overflow-x: hidden;
-  text-overflow: ellipsis;
-  white-space: nowrap;
+  max-width: 400px;
+  margin-bottom: 6px;
+  word-wrap: break-word;
```

Comment:
`word-wrap` is regulated by the `MultilineText`. `max-width` in such a scenario is not necessary

Reply:
@DmitriVanGuard about max-width: in this widget - yes, but in "Emerging trends"  when hover on last icon in row - it causes problems without max-width